src/d3a/d3a_core/simulation.py

```diff
@@ -256,6 +256,11 @@ def _run_cli_execute_cycle(self, slot_resume, tick_resume):
 
     def _update_and_send_results(self, is_final=False):
         self.endpoint_buffer.update_stats(self.area, self.status, self.progress_info)
+        if self.export_on_finish and not self.redis_connection.is_enabled() and is_final:
```

Comment:
```suggestion
        if self.export_on_finish and self.should_export_plots and is_final:
```

Reply:
okay they both have same `not self.redis_connection.is_enabled()` and `self.should_export_plots`, if you prefer the later I would like to rename `should_export_plots ` to something generic as the target here is not to export plots but simply raw market stats.

Reply:
I am definitely in favour of renaming  `should_export_plots` ! I never liked this naming.
How about simply `export_results` ?

Reply:
Can you please rename it everywhere, also in endpoints_buffer please?